robotbase/src/main/java/frc/robot/commands/scoring/SubwooferShot.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot.commands.scoring;
+
+import edu.wpi.first.wpilibj2.command.ParallelDeadlineGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.robot.Constants.SCORING;
+import frc.robot.commands.intake.IntakeFeedToShooter;
+import frc.robot.commands.pivot.PivotHoldAngle;
+import frc.robot.commands.shooter.ShooterSetRPM;
+import frc.robot.commands.shooter.ShooterStop;
+
+public class SubwooferShot extends ParallelDeadlineGroup {
+  public SubwooferShot() {
+    super(
+        new SequentialCommandGroup(
+            new ShooterSetRPM(SCORING.SUBWOOFER_SHOT_SHOOTER_RPMS),
+            new WaitCommand(SCORING.SUBWOOFER_SHOT_WAIT_TO_FIRE_SECONDS),
```

Comment:
You can do wait for rpm here with a timeout. The timeout is a `just in case` thing.